src/test/OpenEXRTest/testDwaLookups.cpp

```diff
@@ -228,8 +228,7 @@ testNoop ()
 
     for (int i = 0; i < 65536; ++i)
     {
-        unsigned short src = (unsigned short) i;
-        assert (src == OPENEXR_IMF_INTERNAL_NAMESPACE::dwaCompressorNoOp[i]);
+        assert ((unsigned short) i == OPENEXR_IMF_INTERNAL_NAMESPACE::dwaCompressorNoOp[i]);
```

Comment:
Should probably just make the loop

for (unsigned short i = 0; ...)

then there's no casting at all

Reply:
fixed